CHANGES.md

```diff
@@ -99,6 +99,8 @@ To be released.
     from `IImmutableList<Uri>` to `IPEndPoint`.
     [[#120], [#123] by Yang Chun Ung, [#126], [#127]]
  -  Added `IStore.ListNamespaces()` method.
+ -  `Transaction` now throws an `InvalidActionTypeException` if an action type
```

Comment:
Although it's trivial, can you refer to the type more specifically, like `Transaction<T>`?  In .NET, two different types of the same name can exist at a time in a namespace if one is generic and other one is non-generic (e.g., [`ImmutableDictionary<TKey,TValue>`][1] and [` ImmutableDictionary`][2]).

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.collections.immutable.immutabledictionary-2
[2]: https://docs.microsoft.com/en-us/dotnet/api/system.collections.immutable.immutabledictionary